contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
   secretName: my-secret
+  configMapName: my-configmap
+  serviceName: my-service
+  osbGuid: b58d64d2-8e01-42b5-a287-bc9ee1860f9d
```

Comment:
In fact, I think allowing them to set it at all is very dangerous because they have no way of knowing if the guid they pass in is unique.